src/js/components/SelectMultiple/SelectMultiple.js

```diff
@@ -421,10 +421,12 @@ const SelectMultiple = forwardRef(
                         {...rest}
                         tabIndex="-1"
                         type="text"
-                        // eslint-disable-next-line max-len
                         placeholder={
+                          // eslint-disable-next-line no-nested-ternary
                           value.length === 0
                             ? placeholder || selectValue || displayLabelKey
+                            : onMore
+                            ? `${value.length} selected`
                             : `${value.length} selected of ${allOptions.length}`
```

Comment:
OK for now, but these should be included in i18n and use MessageContext at some point. Can you file a separate issue for that?

Reply:
Yes, I'll create a separate issue

Reply:
Issue created https://github.com/grommet/grommet/issues/6337